### ISSUE TITLE
Consider faster object detector

### ISSUE THREAD

**Issue by lukeforehand** [lukeforehand/deertracker#23]
MegaDetector works great for gathering training crops, but I'm wondering if it will be too slow for some use cases.  I'd like to try some other models, possibly optimized for mobile device.

**Comment by kbrose** [comment]
I'll be honest I've never trained a bounding box model before, but I do know that "fast" ones exist, e.g. [YOLOv3](https://pjreddie.com/darknet/yolo/).

It might be possible to use the bounding boxes created from the MegaDetector to train its replacement.

**Comment by lukeforehand** [comment]
I didn't realized we'd have to train the detector, I figured I'd try a fast one and see how the classifier works on whatever it finds.  Are you saying that it might not be able to detect all the animals we're looking for ?

**Comment by kbrose** [comment]
It's probably worth experimentation. I'd expect an object detector trained on general categories / images would perform worse than the MegaDetector, but it might still be usable. Most off-the-shelf detectors are trained on the COCO dataset, here are the labels:

```python
COCO_INSTANCE_CATEGORY_NAMES = [
    '__background__', 'person', 'bicycle', 'car', 'motorcycle', 'airplane', 'bus',
    'train', 'truck', 'boat', 'traffic light', 'fire hydrant', 'N/A', 'stop sign',
    'parking meter', 'bench', 'bird', 'cat', 'dog', 'horse', 'sheep', 'cow',
    'elephant', 'bear', 'zebra', 'giraffe', 'N/A', 'backpack', 'umbrella', 'N/A', 'N/A',
    'handbag', 'tie', 'suitcase', 'frisbee', 'skis', 'snowboard', 'sports ball',
    'kite', 'baseball bat', 'baseball glove', 'skateboard', 'surfboard', 'tennis racket',
    'bottle', 'N/A', 'wine glass', 'cup', 'fork', 'knife', 'spoon', 'bowl',
    'banana', 'apple', 'sandwich', 'orange', 'broccoli', 'carrot', 'hot dog', 'pizza',
    'donut', 'cake', 'chair', 'couch', 'potted plant', 'bed', 'N/A', 'dining table',
    'N/A', 'N/A', 'toilet', 'N/A', 'tv', 'laptop', 'mouse', 'remote', 'keyboard', 'cell phone',
    'microwave', 'oven', 'toaster', 'sink', 'refrigerator', 'N/A', 'book',
    'clock', 'vase', 'scissors', 'teddy bear', 'hair drier', 'toothbrush'
]
```

(https://pytorch.org/docs/stable/torchvision/models.html#object-detection-instance-segmentation-and-person-keypoint-detection)
